crates/services/gas_price_service/src/ports.rs

```diff
@@ -12,3 +16,13 @@ pub trait L2Data: Send + Sync {
         height: &BlockHeight,
     ) -> StorageResult<Option<Block<Transaction>>>;
 }
+
+pub trait MetadataStorage: Send + Sync {
+    fn get_metadata(&self, block_height: &BlockHeight)
+        -> Result<Option<UpdaterMetadata>>;
+    fn set_metadata(&mut self, metadata: &UpdaterMetadata) -> Result<()>;
+}
+
+pub trait GasPriceData: Send + Sync {
```

Comment:
the name of the port is `GasPriceData`, but its only function returns a `BlockHeight`.

Would it be possible to have a comment that explains how the two are related?

Reply:
addressed in d51d494f5b

Reply:
The `gas_price_service` crate doesn't know about the `Database<GasPriceDatabase>`=) 


The comment and name of the trait should be more clear regarding what expectation to the type that implements it, and maybe some description why

Reply:
clearer in 64818a6c94